app/src/main/res/values/styles.xml

```diff
@@ -25,4 +25,9 @@
     <style name="SplashTheme" parent="Theme.AppCompat.NoActionBar">
         <item name="android:windowBackground">@drawable/background_splash</item>
     </style>
+
+    <style name="AccentButton" parent="ThemeOverlay.AppCompat.Dark">
+        <item name="colorButtonNormal">@color/grey</item>
+        <item name="colorAccent">@color/colorPrimary</item>
```

Comment:
Why only two attributes? Add all style attribute for the button which are same in sign in and sign up buttons.

Reply:
Done